torchtext/data/example.py

```diff
@@ -9,6 +10,7 @@ class Example(object):
     """
     @classmethod
     def fromJSON(cls, data, fields):
+        warnings.warn('Example class will retire in 0.8.0 release', RuntimeWarning)
```

Comment:
For all of these warnings, the user could ask themselves many questions. Is there any context that should be given? What is the user to do instead? Should the user file an issue if they can't do what they want?

Reply:
We have an issue to explain this change. I might add this to the warning message. And in 0.7.0 release note, I will introduce this again.

Reply:
Would adding the link in the message be a good idea? btw can you share the link here in the comment? :)

Reply:
https://github.com/pytorch/text/issues/664

Reply:
Based on offline discussion: Please use a standard message for all. The message needs to say: (1) users are recommended to move to the new interface by doing abc, but (2) they can keep using the same functionality without the warning by doing xyz.